test/services/resolve/users.test.js

```diff
@@ -9,17 +9,22 @@ describe('resolve/users service', () => {
 		assert.ok(service);
 	});
 
-	it('get error if no scope is found', () => service.get('123').then((_) => {
-		throw new Error('was not supposed to succeed');
-	}).catch((err) => {
-		assert(err.message.includes('No scope found for given id.'));
-		assert(err.name == 'NotFound');
-		assert(err.code == 404);
-	}));
+	it('get error if no scope is found', () =>
+		service
+			.get('123')
+			.then((_) => {
```

Comment:
Codacy found an issue: ['_' is already declared in the upper scope.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)